controllers/vsphereclusteridentity_controller.go

```diff
@@ -61,20 +50,14 @@ var (
 
 // AddVsphereClusterIdentityControllerToManager adds a VSphereClusterIdentity controller to the controller manager.
 func AddVsphereClusterIdentityControllerToManager(ctx context.Context, controllerManagerCtx *capvcontext.ControllerManagerContext, mgr manager.Manager, options controller.Options) error {
-	var (
-		controllerNameShort = fmt.Sprintf("%s-controller", strings.ToLower(identityControlledTypeName))
-		controllerNameLong  = fmt.Sprintf("%s/%s/%s", controllerManagerCtx.Namespace, controllerManagerCtx.Name, controllerNameShort)
-	)
-
 	reconciler := clusterIdentityReconciler{
 		ControllerManagerCtx: controllerManagerCtx,
 		Client:               controllerManagerCtx.Client,
-		Recorder:             record.New(mgr.GetEventRecorderFor(controllerNameLong)),
+		Recorder:             mgr.GetEventRecorderFor("vsphereclusteridentity-controller"),
```

Comment:
Recorders are largely unused in CAPV. I just thought I keep them around instead of deleting them everywhere.

But I could also have deleted a bunch of them.

Reply:
ok